crates/common/src/meta.rs

```diff
@@ -44,6 +69,37 @@ impl TryDecodeRainlangSource for String {
     }
 }
 
+impl TryDecodeDotrainInstance for String {
```

Comment:
same for this - could it go upstream?

Reply:
Done, but now it is a method:
`UnpackedMetadata::parse_from_hex(&str) -> Result<Vec<UnpackedMetadata>, Error>`